webapp/publisher/snaps/views.py

```diff
@@ -1351,6 +1351,12 @@ def get_snap_builds(snap_name):
         store_name=details["snap_name"],
     )
 
+    github = get_github_api(flask.session.get("github_auth_secret"))
```

Comment:
Sorry... this is actually pointless. For some reason I thought the instantiation of `GitHubAPI` would be a long thing, and so it would be too verbose to do it in-place in every view function.

But actually, if it's just `github = GitHubAPI(flask.session.get("github_auth_secret"))`, you should just do it directly inline here, and not bother with the factory function.

Reply:
It's fine, it makes sense, at the beginning this PR was like this but without the token argument.

I just did this change.